skyvern/webeye/scraper/domUtils.js

```diff
@@ -601,6 +618,21 @@ function buildTreeFromBody() {
     } else if (attrs["role"] && attrs["role"].toLowerCase() === "listbox") {
       // if "role" key is inside attrs, then get all the elements with role "option" and get their text
       selectOptions = getListboxOptions(element);
+    } else if (isComboboxDropdown(element)) {
+      // open combobox dropdown to get options
+      element.click();
+      const listBox = document.getElementById(
```

Comment:
is it possible that this listbox will be undefined / null? shall we handle that

Reply:
Yeah, you're right. We should handle the null exception here, tho the possibility to be null is very low.